woocommerce/woocommerce-android
# Issue/3472 fix first day of week

Fixes #3472 

***Ready for review, but not ready for merge***
~1. Wait until https://github.com/woocommerce/woocommerce-android/pull/3694 gets merged into develop~
2. Wait until https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1926 gets merged into develop
3. Update FluxC hash with a tag
4. Remove "Not ready for merge" label
5. Merge this PR

Graph for "This week" on the dashboard always display "Sunday" as the first day of week. Ideal solution would use "Week starts on" settings from WPAdmin as suggested [here](https://github.com/woocommerce/woocommerce-android/issues/3472#issuecomment-799788963). However, AFAICT this solution is not straighforward. We decided to let the OS choose the first day of week based on user's locale.

To test:
1. Set your device to English (US)
2. Restart the Woo app
3. Open dashboard
4. Click on "This week" tab
5. Notice the week starts with Sunday
6. Set your device for example to French
2. Restart the Woo app
3. Open dashboard
4. Click on "This week" tab
5. Notice the week starts with Monday 

Update release notes:
